README.md

```diff
@@ -23,7 +23,7 @@ your project:
 #### Gradle
 
 ```groovy
-implementation "com.shopify:checkout-sheet-kit:3.0.1"
+implementation "com.shopify:checkout-sheet-kit:3.0.4"
```

Comment:
Would this work?

```suggestion
implementation "com.shopify:checkout-sheet-kit:3"
```

Reply:
I think it'd be pretty rare for somebody to do this